language/en_us/domains_tlds.php

```diff
@@ -13,9 +13,11 @@
 $lang['DomainsTlds.!error.email_forwarding.valid'] = 'Invalid value for Email Forwarding, must be 0 or 1.';
 $lang['DomainsTlds.!error.id_protection.valid'] = 'Invalid value for ID Protection, must be 0 or 1.';
 $lang['DomainsTlds.!error.epp_code.valid'] = 'Invalid value for EPP Code, must be 0 or 1.';
+$lang['DomainsTlds.!error.unsupported_tld'] = 'The provided TLD is unsupported by the selected registrar module.';
```

Comment:
The language key for this message should follow the standard naming schema 'DomainsTlds.!error.module_id.supported'